teamwork/apps/projects/views/EditProjectView.py

```diff
@@ -284,11 +313,12 @@ def add_member(request, slug, uname):
 
     adjust_pendinglist(request, project, mem_to_add)
 
+    return True # Success
```

Comment:
Assuming we're doing some kind of boolean comparison somewhere in the code?

Reply:
Yeah we check if the user was successfully added. If they were we send an email to notify them.